### DISCUSSION TITLE
请问事件选项选错了如何修改

### DISCUSSION THREAD

**Discussion by chenzyo** [NateScarlet/auto-derby#111] [Q&A] [answered]
有个俩选项的事件我手滑选了4，导致每次遇到这事件都会卡住。   我把single_mode_event_images.local目录里对应的截图删除也没用，  请问还有办法修改吗？

**Comment by NateScarlet** [comment, marked as answer]
single_mode_choices.csv 里面删

**Comment by chenzyo** [comment]
谢谢， 我找找
